Sources/DeepLearning/Initializers.swift

```diff
@@ -13,9 +13,295 @@
 // limitations under the License.
 
 #if !COMPILING_TENSORFLOW_MODULE
-@_exported import TensorFlow
+import TensorFlow
 #endif
 
+public extension Tensor {
+    /// Creates a tensor with the specified shape and a single, repeated scalar
+    /// value.
+    ///
+    /// - Parameters:
+    ///   - shape: The dimensions of the tensor.
+    ///   - repeatedValue: The scalar value to repeat.
+    @inlinable
+    @available(*, deprecated, renamed: "init(repeating:shape:)")
+    init(shape: TensorShape, repeating repeatedValue: Scalar) {
+        self.init(repeating: repeatedValue, shape: shape)
+    }
+
+    /// Creates a tensor with the specified shape and a single, repeated scalar value.
+    ///
+    /// - Parameters:
+    ///   - repeatedValue: The scalar value to repeat.
+    ///   - shape: The dimensions of the tensor.
+    @inlinable
+    @differentiable(
+        vjp: _vjpInit(repeating:shape:) where Scalar : TensorFlowFloatingPoint)
+    init(repeating repeatedValue: Scalar, shape: TensorShape) {
+        self = Raw.fill(
+            dims: Tensor<Int32>(shape.dimensions.map(Int32.init)),
+            value: Tensor(repeatedValue))
+    }
+
+    /// Creates a tensor by broadcasting the given scalar to a given rank with
+    /// all dimensions being 1.
+    @inlinable
+    // @differentiable(where Scalar : TensorFlowFloatingPoint)
+    init(broadcasting scalar: Scalar, rank: Int) {
+        self = Tensor(scalar).reshaped(to: TensorShape(repeating: 1, count: rank))
+    }
+
+    /// Creates a tensor of shape `[4]` from a 4-tuple.
+    /// - Note: This is intended for internal use, for example, to initialize a
+    ///   tensor attribute from `convolved2D`'s `strides` argument.
+    @inlinable
+    internal init(_ scalars: (Scalar, Scalar, Scalar, Scalar)) {
+        self.init([scalars.0, scalars.1, scalars.2, scalars.3])
+    }
+}
+
+internal extension Tensor where Scalar : TensorFlowFloatingPoint {
+    @inlinable
+    static func _vjpInit(
+        repeating repeatedValue: Scalar,
+        shape: TensorShape
+    ) -> (Tensor, (Tensor) -> Scalar) {
+        return (Tensor(repeating: repeatedValue, shape: shape), {
+            $0.sum().scalarized()
+        })
+    }
+}
+
+//===------------------------------------------------------------------------------------------===//
+// Casting
+//===------------------------------------------------------------------------------------------===//
+
+public extension Tensor where Scalar : Numeric {
```

Comment:
In the swift-apis library we are no longer using the stdlib code styles but the Google Swift Style Guide instead (except for 4-space indentation). One difference is that there's no space before any colon, including type inheritance clauses. 
```suggestion
public extension Tensor where Scalar: Numeric {
```